web/templates/single_message.hbs

```diff
@@ -11,6 +11,7 @@
     <div class="messagebox">
         <div class="messagebox-content {{#if status_message}}is-me-message{{/if}}">
             {{> message_body}}
+            {{!-- message_edit_form.hbs will be dynamically added here. --}}
```

Comment:
Added the comment mentioned in https://github.com/zulip/zulip/pull/31478#discussion_r1736962733 over here.

Reply:
Mentioned `message_edit_form.hbs` instead of just `message_edit_form` to avoid confusion from the class of the same name `.message_edit_form`.

Reply:
Changed to "message_edit_form.hbs is inserted here when editing a message." before merging to be a bit more precise.